components/widgets/inputs/channels_input.tsx

```diff
@@ -107,17 +107,13 @@ export default class ChannelsInput extends React.PureComponent<Props, State> {
         const Msg: any = components.NoOptionsMessage;
         return (
             <div className='channels-input__option channels-input__option--no-matches'>
-                <FormattedMarkdownMessage
```

Comment:
For this change and the one in users_emails_input, I don't know if these used to work before since `FormattedMarkdownMessage` never supported the function-as-a-child pattern that `FormattedMessage` does. I'm not sure if this works now, but it seems more likely to work than the old way.

@neallred I know you weren't the one who wrote this code, but since you seem to be the last person who touched it, do you happen to know where this is used? I wasn't able to test it myself since I don't know where these appear